source/common/stats/symbol_table_impl.h

```diff
@@ -132,13 +132,15 @@ class SymbolTableImpl : public SymbolTable {
   bool lessThan(const StatName& a, const StatName& b) const override;
   void free(const StatName& stat_name) override;
   void incRefCount(const StatName& stat_name) override;
-  SymbolTable::StoragePtr join(const std::vector<StatName>& stat_names) const override;
+  StoragePtr join(const std::vector<StatName>& stat_names) const override;
   void populateList(const absl::string_view* names, uint32_t num_names,
                     StatNameList& list) override;
   StoragePtr encode(absl::string_view name) override;
   void callWithStringView(StatName stat_name,
                           const std::function<void(absl::string_view)>& fn) const override;
 
+  void encode(absl::string_view name, Encoding& encoding);
```

Comment:
Is this implemented?

Reply:
no it was detritus; deleted. good catch.